homeassistant/components/jvc_projector/coordinator.py

```diff
@@ -21,8 +21,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-INTERVAL_SLOW = timedelta(seconds=60)
-INTERVAL_FAST = timedelta(seconds=6)
+INTERVAL_SLOW = timedelta(seconds=10)
+INTERVAL_FAST = timedelta(seconds=5)
```

Comment:
IMO we should leave SLOW at 60s. Is there a use case for it being faster?

I agree with making FAST faster. In fact maybe even 3s? **Or** we could even consider making it 1s when the power state is COOLING or WARMING, and leave it at 6s for ON. We really only care about fast updates during the cooling and warming transitions I assume.

Reply:
@SteveEasley yes, there is a reason. When in off or standby with the 60 sec the projector is checked every 1 minute. This is bad for all "power on" automations.
In my case when the projector is powered on and enters in the warming status status I have to power on the media player, the amplifier, open the curtain, close the shadows. All of this must be finished before the projector gets out of warming and is on... and this takes less than 1 minute.
So If we check in SLOW every 1 minute in 80% of the cases the projector is on but the automation has not started, and is not good.

For going even Faster in Fast the documentation of the projector explains that there should be a little time between the different phases of the handshake process and between commands so I think 5 is good. When I tested in on 2 or 3 I got some random issues. 

That's why I suggested 5 - 10 if we want to have a fast/slow... in  my personal integration I had everything on 5 :) so no change

Reply:
I guess I am still a little confused here on the SLOW side. When the projector is turned on it goes into WARMING, that would start the FAST refreshes. Why would other automations be waiting for a minute on it? I might be missing something obvious though.

5s sounds good to me on FAST then. Sounds like you have tested it well.

Reply:
Ok I just read your [Issue comment](https://github.com/home-assistant/core/issues/109732#issuecomment-1937095664) and that provided me the context I was missing. You want the projector power state to drive automations. Not really understanding your concern of it being risky to let HA control your projector, but agree there is no real negative cost to increasing it to 10s. So works for me.

Thanks for all the time you have spent on this. It was on my TODO :)